Create Mentorship Winter 2021 Signups Newspost (#4521)
ppy/osu-wiki: news/2020-11-26-community-mentorship-program-winter-2021-signups-now-open.md

---
layout: post
title: Community Mentorship Program Winter 2021 Signups Now Open
date: 2020-11-26 08:00:00 +0000
---

> With holiday cheer and a upcoming new year, the 17th cycle of the Community Mentorship program is on its way! If you've ever found yourself wanting to expand your current knowledge and skills or help beginner mappers get started, this program will serve as a great opportunity for you!

Reviewer: ```suggestion
With holiday cheer and an upcoming new year, the 17th cycle of the Community Mentorship program is on its way! If you've ever found yourself wanting to expand your current knowledge and skills or help beginner mappers get started, this program will serve as a great opportunity for you!
```

![](/wiki/shared/news/banners/community-mentorship-program.jpg)

> The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills under the guidance of more experienced mappers and modders in the community. 

Reviewer: ```suggestion
The Community Mentorship program serves as a place for upcoming mappers and modders to develop their skills under the guidance of more experienced mappers and modders in the community. 
```

> The Community Mentorship program serves as a place for upcoming mappers and modders to develop their skills under the guidance of more experienced mappers and modders in the community. 

Reviewer 2: ```suggestion
The Community Mentorship program serves as a place for upcoming creators to develop their skills under the guidance of more experienced mappers and modders from the community. 
```

Author: Generalizing like that makes things unnecessarily broad imo. "creator" can be interpreted beyond mapper/modder eg. storyboarder or skinner, which we don't cater to.

Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events (like mapping contests), hang out with fellow mentees, and more!

> If you're experienced in mapping and have a desire to take those who are new under your wing, then give joining us as Mentor a shot! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1188335), you'll be able to apply right away. If you're only just getting into mapping or are eager to hone your skills, then signing up as Mentee might just be your way to do it!

Reviewer 2: ```suggestion
If you're experienced in mapping and have a desire to take those, who are new, under your wing, then give joining us as a Mentor a shot! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1188335), you'll be able to apply right away. If you're only just getting into mapping or are eager to hone your skills, then signing up as a Mentee might just be your way to do it!
```

Author: Really odd comma placement imo. Reads like "those" is a group of people on its own and "who are new" is a specification for them, even though "those" would have nothing to refer to and loses its meaning. Should together stay as is.
As Mentor/Mentee reads more naturally to me ? Should be okay.

---

For this following cycle, the timeline is as follows (all opening and closing dates start and end at midnight UTC+0):

> - Mentor signups will open on November 30th and close on December 6th.
> - Mentee signups will open on December 27th and close on January 3rd.
> - The 2021 Winter cycle will run from January 4th to April 9th.

Reviewer: ```suggestion
- Mentor signups will open on **November 30th** and close on **December 6th**.
- Mentee signups will open on **December 27th** and close on **January 3rd**.
- The 2021 Winter cycle will run from **January 4th** to **April 9th**.
```
just thought these start/end dates could use some bold formatting, feel free to dismiss this one though - just a suggestion

> Throughout this active cycle time, the organisation team will host various events on the Mentorship Discord server, engaging and offering events for mentees and mentors to participate in. These include feedback lobbies, mapping contests, and more! Some of our previous mapping puzzles can be found here at this [thread](https://osu.ppy.sh/community/forums/topics/565382), and previous discussions can be found [here](https://osu.ppy.sh/community/forums/topics/514122).

Reviewer: ```suggestion
Throughout this active cycle time, the organization team will host various events on the Mentorship Discord server, engaging and offering events for mentees and mentors to participate in. These include feedback lobbies, mapping contests, and more! Some of our previous mapping puzzles can be found here in this [thread](https://osu.ppy.sh/community/forums/topics/565382), and previous discussions can be found [here](https://osu.ppy.sh/community/forums/topics/514122).
```
also `events in the Mentorship Discord server` sounds better but not sure

> Throughout this active cycle time, the organization team will host various events on the Mentorship Discord server, engaging and offering events for mentees and mentors to participate in. These include feedback lobbies, mapping contests, and more! Some of our previous mapping puzzles can be found here in this [thread](https://osu.ppy.sh/community/forums/topics/565382), and previous discussions can be found [here](https://osu.ppy.sh/community/forums/topics/514122).

Reviewer 2: ```suggestion
Throughout this active cycle time, the organization team will host various events on the Mentorship Discord server, engaging and offering events for mentees and mentors to participate in. These include feedback lobbies, mapping contests, and more! Some of our previous mapping puzzles can be found here in [this thread](https://osu.ppy.sh/community/forums/topics/565382), and previous discussions can be found [here](https://osu.ppy.sh/community/forums/topics/514122).
```
